deephyper/skopt/moo/_multiobjective.py

```diff
@@ -0,0 +1,145 @@
+import abc
+from unicodedata import is_normalized
+import numpy as np
+from deephyper.skopt.utils import is_listlike
+
+
+class MoScalarFunction:
```

Comment:
should be a subclass of `abc.ABC` if abstract such as
```python
class MoScalarFunction(abc.ABC):
  ...
```

Reply:
Done. Thanks for the suggestion!